### PR TITLE
docs(contributing/callbacks): fix links and formatting

### PR DESCRIPTION
**Pull request by alexcg1**
Some small fixes:
- CONTRIBUTING - fix docs example links, remove "Alex" stuff
- callbacks - add backticks around relevant text
- README - fix links

- [X] check and update documentation. See [guide](https://github.com/jina-ai/jina/blob/master/CONTRIBUTING.md#-contributing-documentation) and ask the team.

### CODE REVIEW

File: docs/proto/docs.md [hunk 1]
```diff
@@ -0,0 +1,398 @@
+# Protocol Documentation
+<a name="top"></a>
+
```
Comment (samsja) [inline docs/proto/docs.md:3]: Why the modification on the proto docs ? It looks like the file was commited by mistake or is from an old version

Comment (alexcg1) [reply docs/proto/docs.md:3]: Damn, looks like a mistake. I'll try to fix

Comment (alexcg1) [reply docs/proto/docs.md]: Removed that commit. For some reason GitHub is saying I re-added all the other commits again :shrug:

**Comment by alexcg1** [comment]
Hmmm...k8s ci constantly timing out

**Review by alaeddine-13 — DISMISSED** [review]

**Comment by alexcg1** [comment]
Can I get another approval? Then I think we can merge

@samsja @JohannesMessner

**Review by samsja — APPROVED** [review]

**Comment by alexcg1** [comment]
@hanxiao can you review?

**Pull request merged**